python/tests/test_graphdb/test_graphdb_imports.py

```diff
@@ -56,3 +56,221 @@ def test_import_with_int():
     g2.import_node(g.node(1))
     g2.import_nodes([g.node(2), g.node(3)])
     assert g2.count_nodes() == g.count_nodes()
+
```

Comment:
If you can take the comments I made within the rust tests and also apply them here that would be great